.eslintrc.js

```diff
@@ -92,6 +92,7 @@ module.exports = {
     "@typescript-eslint/require-await": "off",
     "@typescript-eslint/no-unused-vars": "off",
     "unused-imports/no-unused-imports": "error",
+    "@typescript-eslint/no-non-null-assertion": "off",
```

Comment:
what is the reason? This to me looks like a very sensible rule

Reply:
it just gives warnings by default, and the non-null-assertions are needed even in obvious situations when it should have a value. because we have many of these warnings I've turned this off because it just had made the debug warning console messy and I hadn't been able to keep track with the new significant static errors.